python/.noserc

```diff
@@ -1,12 +0,0 @@
-# this file's explicitly loaded from setup.cfg (.noserc isn't a
-# standard config path), separated out due to '%(...)s'
```

Comment:
While I appreciate that this file is no longer needed, due to dropping `nose`, we should probably port the various [config options](https://docs.pytest.org/en/stable/reference/reference.html#configuration-options) (where appropriate) so that things remain consistent.

(e.g we update `pytest.ini` to include 
```
log_format = %(asctime)s,%(msecs)03d %(name)s: %(levelname)s: %(message)s
log_file_date_format = %H:%M:%S
```
etc.

Reply:
Added those!